exercise_05/isAdditionGreaterThanFifty.js

```diff
@@ -4,3 +4,15 @@
  * input: number (summand2)
  * output: boolean
  */
+
+/**
+ * returns true if numbers sum is 50 or greater than 50
+ * @param {*} summand1 
+ * @param {*} summand2 
+ * @returns 
+ */
+const isAdditionGreaterThanFifty = function (summand1, summand2) {
+    return summand1 + summand2 >= 50;
```

Comment:
Avoid magic numbers. 

```suggestion
    return (summand1 + summand2) >= THRESHOLD;
```

- [ ] Extract number to constant